packages/cli/lib/commands/watch.js

```diff
@@ -105,6 +106,9 @@ async function command(src, argv) {
 		argv.sw = toBool(argv.sw);
 	}
 
+	let cwd = resolve(argv.cwd);
+	require('dotenv').config({ path: resolve(cwd, '.env') });
```

Comment:
```suggestion
	require('dotenv').config();
```

Wouldn't this work as well?
https://github.com/motdotla/dotenv#path

Reply:
In normal circumstances, sure.

However, that would break in our test suite (and probably mono-repos too): `cwd` would differ from the project root.

I ran into that issues and moved the dotenv setup here accordingly.